assets/src/userTestGroups.ts

```diff
@@ -4,6 +4,8 @@ import appData from "./appData"
 const TestGroups = array(string())
 type TestGroups = Infer<typeof TestGroups>
 
+export const MAP_BETA_GROUP_NAME = "map-beta"
```

Comment:
comment: I'm really glad you 'symbolized' this! 
I figured it might just be bikeshedding to ask what we should name the test group.

Reply:
Yeah it is a fair question! I'm not sure what an appropriately descriptive name would be, but have a question for product in the ticket & will update if there is a change requested!